### PR TITLE
Stateless cloud interface

### PR DESCRIPTION
**Pull request by hokwangchoi**
Now, cloud is available for stateless environment.
Changes are made to clarify the name and the information about the cloud.

### CODE REVIEW

File: ddln_imaginator.proto [hunk 1]
```diff
@@ -63,8 +60,9 @@ message Environment {
         // density value, typical range 0..1 [-]
         optional float density = 3;
 
-        // directional x, y, z values to set the wind in ENU coordinates system [m/s]
-        optional Point3d velocity = 4;
+        // directional x, y values for displacement in ENU coordinates system [m] and z value for shape changing
```
Comment (PDLdeLange) [inline ddln_imaginator.proto]: If possible, describe unit for z as well.

**Review by PDLdeLange — APPROVED** [review]

File: ddln_imaginator.proto [hunk 2]
```diff
@@ -63,8 +60,9 @@ message Environment {
...
+        // effect(curling up). 'z' value does not affect the altitude of the cloud.
```
Comment (gyakushev) [inline ddln_imaginator.proto]: don't use `Point3d` then: use separate `x, y, curling` variables

File: ddln_imaginator.proto [hunk 3]
```diff
@@ -63,8 +60,9 @@ message Environment {
...
+        optional Point3d displacement = 4;
```
Comment (gyakushev) [inline ddln_imaginator.proto]: use different identifier from deprecated `velocity` (e.g. 5)

**Review by gyakushev — CHANGES_REQUESTED** [review]
Let's try to avoid having to remove fields in the future.

File: ddln_imaginator.proto (hunk 1 above)
Comment (hokwangchoi) [reply ddln_imaginator.proto]: It will be all in meters.

**Review by gyakushev — APPROVED** [review]

**Pull request merged**